src/Connection.cpp

```diff
@@ -6,6 +6,9 @@
 #include "murmur_pch.h"
 
 #ifdef Q_OS_WIN
+#	ifdef _INC_QOS2
```

Comment:
Why? What error did you get?

qos2.h is already included via murmur_pch.h.... Was this a redefinition error... Or something else?